internal/venice-common/src/main/java/com/linkedin/venice/pubsub/manager/TopicManager.java

```diff
@@ -801,6 +802,18 @@ public String getPubSubClusterAddress() {
     return this.pubSubClusterAddress;
   }
 
+  /**
+   * Resolve the position bytes to a PubSubPosition object.
+   * @param topicPartition the topic partition to resolve the position for
+   * @param positionBytes the position bytes to resolve
+   * @return the resolved PubSubPosition object
+   */
+  public PubSubPosition resolvePosition(PubSubTopicPartition topicPartition, byte[] positionBytes) {
```

Comment:
Is this not used anywhere? This null looks fishy If not let's remove it for now and add back when it is ready?